src/utils.ts

```diff
@@ -10,8 +10,8 @@ export function mapToKeys<T>(collection: Optional<T[]>) {
 // wraps urijs to handle edge cases that would normally error out or cause unexpected behavior
 // in all funcs set a default empty string, since URI.func() !== URI.func(undefined) [<- undefined throw error in most cases]
 // in all funcs set a default empty string, since URI.func('') will reset fields back to null while URI.func() will not
-export function URI(url: string | uri.URI = '') {
-  const uri: uri.URI = !url || typeof url === 'string' ? new URIJS(url) : url;
+export function URI(url: string | URI = '') {
+  const uri: URI = !url || typeof url === 'string' ? new URIJS(url) : url;
```

Comment:
I do not know why TypeScript 3.7 was assuming this was valid. It was not.

Reply:
Assuming what was valid? The code looks unchanged - just typings changed `uri.URI` -> `URI`.

Reply:
`uri.URI` was reported as undefined — and effectively, nothing is importing `uri` in such file. I do not know where that's coming from.